src/lib/profile/experience/validate-academic-experience.ts

```diff
@@ -0,0 +1,41 @@
+import { object, string, boolean, date, ref } from 'yup'
+
+const schema = object({
+  organizationName: string().required('Campo requerido'),
+  jobTitle: string().required('Campo requerido'),
+  description: string().required('Campo requerido'),
+  entryDate: date().nullable().default(null).required('Campo requerido'),
+  departureDate: date()
+    .nullable()
+    .transform((curr: Date, orig) => (orig === '' ? null : curr))
+    .when('entryDate', {
+      is: (entryDate: Date) => entryDate !== null,
+      then: (s) => s.min(ref('entryDate'), 'La fecha de salida debe ser posterior a la de entrada')
+    }),
+
+  freelancer: boolean(),
```

Comment:
```suggestion
    }),
  freelancer: boolean().default(false),
```
El valor por defecto debiera ser false, por interfaz

Reply:
Eso lo que haría es que si pasas la propiedad freelancer como undefined al ```.validate()``` entonces se validaría como correcto, lo cual no tiene sentido en nuestro caso. Lo que sí le veo sentido ahora que me doy cuenta, es a colocarle un ```.required()```